providers/tests/openlineage/extractors/test_manager.py

```diff
@@ -313,6 +313,7 @@ def use_read():
 
     dr = dag_maker.create_dagrun()
     ti = TaskInstance(task=task, run_id=dr.run_id)
+    ti.refresh_from_db()
```

Comment:
The reason for this is explained in [**Issues identified Point (1)**](https://github.com/apache/airflow/pull/43243#issuecomment-2439774584)